### ISSUE TITLE
Confusion of linear classifier/evaluation

### ISSUE THREAD

**Issue by HymEric** [google-research/simclr#52]
Hi, it's a great work and I have read the paper. But I'm confused about the classification phase. 

1) First, do self-supervised training of contractive learning with simCLR framework (means we get a good extractor to extract features).
2.) Then fixe the extractor params (means the f(.) in the paper) and get the features for supervised training in the classification task with a linear classifier. 

In 2), is this mean the linear evaluation? And **in this phase what's the linear classifier**, linear SVM, the linear logistic regression or the one-layer network with softmax?

I appreciate your reply.

**Comment by chentingpc** [comment]
In 2) there are two ways for use the pre-trained model: one is fine-tuning where you can change all parameters, the other is linear evaluation, where you add a linear layer with softmax on top while fixing f(.), the linear layer is trained using SGD.
https://github.com/google-research/simclr/blob/01ddaf0bd692ee945dad7ff5fb07b26df1b9edbe/model.py#L84

**Comment by HymEric** [comment]
Thanks a lot. I understand.
